config/locales/publishers.yml

```diff
@@ -199,7 +199,29 @@ en:
           heading: Shortlist applicant
           page_title: Shortlist applicant
         show:
+          application_sections:
+            heading: Application sections
+          ask_for_support:
+            heading: Ask for support if you have a disability or other needs
+          current_role_and_employment_history:
+            heading: Current role and employment history
+            gaps_in_employment:
+              heading: Gaps in employment
+          declarations:
+            heading: Declarations
+          education_and_qualifications:
+            heading: Education and qualifications
```

Comment:
Shouldn't we reuse the headings from `"jobseekers.job_applications.build.#{step}"` so that we only have to change them in one place?

Reply:
I guess so if we never think they're never going to be different?